HelpSource/Classes/CondVar.schelp

```diff
@@ -0,0 +1,415 @@
+class::CondVar
```

Comment:
It would be good to explain why this is a "variable", but unlike e.g. a `FlowVar` it doesn't hold a value.

Reply:
yes, good point, I'm not quite sure where the name "condition variable" comes from as they aren't any more variable-like to me than other synchronization objects like locks, mutexes, semaphores, etc.

Reply:
would this wording work for you? "Although it has "variable" in the name, a condition variable doesn't wrap a variable like, for example, FlowVar does."

Reply:
Yes, that would be good! While popular, it still is somewhat a misnomer.

Reply:
done!